pandas/core/internals/blocks.py

```diff
@@ -1338,7 +1338,28 @@ def quantile(
         return new_block(result, placement=self._mgr_locs, ndim=2)
 
 
-class ExtensionBlock(Block):
+class EABackedBlock(Block):
```

Comment:
2 bad you have to do this (create a nother block type), any better way (e.g. handle it in the EA impl)?

Reply:
there are actually a few other methods im planning to move here to share between ExtensionBlock/NDArrayBackedExtensionBlock.  I think the code de-duplication makes this a net plus